CITATION.rst

```diff
@@ -1,13 +1,27 @@
+Citation Guidelines
+===================
+
+|JOSS| |Zenodo|
+
+
 If you use SkyPy for work or research presented in a publication (whether
-directly, or as a dependency to another package) we recommend and encourage
+directly, or as a dependency of another package) we recommend and encourage
 the following acknowledgment:
 
   This research made use of SkyPy, a Python package for forward modeling
```

Comment:
```suggestion
.. admonition:: Acknowledgement

  This research made use of SkyPy, a Python package for forward modeling
```

Just an idea; I think this improves the html docs but doesn't display well in GitHub. What do people think?

Reply:
I like it.

Reply:
I like it

Reply:
Readthedocs looks good: https://skypy--496.org.readthedocs.build/en/496/project/citation.html
GitHub not so good: https://github.com/Lucia-Fonseca/skypy/blob/citation/CITATION.rst
While I like how the documentation looks and think it is more important, I don't know if people are happy with the ugly rendering on GitHub

Reply:
In my opinion, it does not look that ugly in GitHub but let's hear people's opinions.

Reply:
I find a `pull-quote` more apropos here, and it should render nicely in either form.

Reply:
Do you mean this?
```suggestion
  .. pull-quote::

  This research made use of SkyPy, a Python package for forward modeling
  astronomical surveys (Amara et. al., 2021, SkyPy Collaboration, 202x).
```